Strange code in all the platfotm_conf.h files always enabled BUILD_RPC,
even when it is disabled.  This is harmless since the avr32 linker garbage
collects the unreferenced functions.  It just provoked a few compiler
warnings when LUA_DUMP (ie undump) is not defined.

Index: elua-svn/src/platform/avr32/MIZAR32/mizar32_conf.h
===================================================================
--- elua-svn.orig/src/platform/avr32/MIZAR32/mizar32_conf.h	2010-12-02 10:34:18.097727002 +0100
+++ elua-svn/src/platform/avr32/MIZAR32/mizar32_conf.h	2010-12-02 10:59:43.473727002 +0100
@@ -51,9 +51,6 @@
 #define RPCLINE _ROM( AUXLIB_RPC, luaopen_rpc, rpc_map )
 #else
 #define RPCLINE
-#if !defined( BUILD_RPC )
-#define BUILD_RPC
-#endif
 #endif
 
 #define LUA_PLATFORM_LIBS_ROM\
